routers/repo/setting_protected_branch.go

```diff
@@ -186,6 +196,13 @@ func SettingsProtectedBranchPost(ctx *context.Context, f auth.ProtectBranchForm)
 		if strings.TrimSpace(f.MergeWhitelistTeams) != "" {
 			mergeWhitelistTeams, _ = base.StringsToInt64s(strings.Split(f.MergeWhitelistTeams, ","))
 		}
+		if strings.TrimSpace(f.MergeWhitelistTeams) != "" {
```

Comment:
Isn't this the same thing as the `if` just above?

Reply:
This has been removed.